docs/FAQ.md

```diff
@@ -82,7 +82,7 @@ Remember this is a free/open community support. We may not be available 24/7 to
 
 - **Question 8**: CPU are not kept busy (like 10% CPU usage only) in Windows when using LightGBM on very large datasets with many core systems.
 
-- **Solution 8**: Please use [use Visual Studio](https://github.com/Microsoft/LightGBM/issues/740) as it may be 10x faster than MinGW especially for very large trees.
+- **Solution 8**: Please [use Visual Studio](https://github.com/Microsoft/LightGBM/issues/740) as it may be 10x faster than MinGW especially for very large trees.
```

Comment:
All other links with brand names in docs leads to third party sites, but this one refers to GitHub issue. So, I suppose, this will be confusing for users.